media-radio/ax25-apps/ax25-apps-0.0.8_rc5-r1.ebuild

```
@@ -0,0 +1,51 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+MY_P="${P/_/-}"
+
+DESCRIPTION="Basic AX.25 (Amateur Radio) user tools, additional daemons"
+HOMEPAGE="http://www.linux-ax25.org"
+SRC_URI="http://www.linux-ax25.org/pub/${PN}/${MY_P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 ppc x86"
+
+RDEPEND="
+	>=dev-libs/libax25-0.0.12_rc2:=
+	sys-libs/ncurses:=
+	virtual/pkgconfig
+	!dev-ruby/listen
+"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+src_prepare() {
+	eapply_user
+	# fix missing prototype for malloc
+	sed -i -e "/^#include /i #include <stdlib.h>" ax25ipd/routing.c || die
+	eapply "${FILESDIR}"/${PN}-0.0.8_rc5-tinfo.patch
```

In general it would be better to call `eapply_user` or `default´ *after* our own changes.

```suggestion
	# fix missing prototype for malloc
	sed -i -e "/^#include /i #include <stdlib.h>" ax25ipd/routing.c || die
	eapply "${FILESDIR}"/${PN}-0.0.8_rc5-tinfo.patch
	eapply_user
```